### PR TITLE
chore(doc): add setup section

### PR DESCRIPTION
**Pull request by MR2011**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,23 @@ For a detailed understanding of the system's architecture and design, refer to t
 
 ## Requirements and Setup
```
Comment (drochow) [inline README.md:42]: I like the description, but I think the default way should be using the Makefile no? 

Also I think we need to add a couple more information such as environment variables etc.  

WDYT?

Comment (MR2011) [reply README.md:42]: You're right. I'll add this

File: README.md [hunk 2]
```diff
@@ -40,7 +40,60 @@ For a detailed understanding of the system's architecture and design, refer to t
...
+DB_NAME=heureka
+DB_ADDRESS=localhost
+DB_PORT=3306
+DB_SCHEMA=internal/database/mariadb/init/schema.sql
+
+DB_CONTAINER_IMAGE=mariadb:latest
+
+DOCKER_IMAGE_REGISTRY=hub.docker.com
+
+DOCKER_CREDENTIAL_STORE=docker-credential-desktop
```
Comment (drochow) [inline README.md:61]: I think this is not a self-explanatory env var, lets add some context with a comment?

File: README.md [hunk 3]
```diff
@@ -40,7 +40,60 @@ For a detailed understanding of the system's architecture and design, refer to t
...
+
+LOG_PRETTY_PRINT=true
+
+LOCAL_TEST_DB=true
```
Comment (drochow) [inline README.md:65]: I think this is not a self-explanatory env var

File: README.md [hunk 4]
```diff
@@ -40,7 +40,60 @@ For a detailed understanding of the system's architecture and design, refer to t
...
+
+SEED_MODE=false
```
Comment (drochow) [inline README.md:67]: I think this is not a self-explanatory env var

**Review by drochow — CHANGES_REQUESTED** [review]

**Review by drochow — APPROVED** [review]

**Pull request merged**